app/models/exam_version.rb

```diff
@@ -510,4 +541,20 @@ def bottlenose_summary(with_names: true)
       end.compact
     end
   end
+
+  private
+
+  def duration_valid
+    return unless effective_duration > effective_time_window
+
+    mins = (effective_time_window / 60.0).to_i
```

Comment:
I was going to suggest `.minutes`, but then...
![image](https://user-images.githubusercontent.com/6677292/190271248-73ac8390-1794-4aca-b43e-43cad99baf3c.png)
why?!

Reply:
Time is a mystery to us all.